servicetalk-data-protobuf-jersey/src/test/java/io/servicetalk/data/protobuf/jersey/resources/SingleProtobufResources.java

```diff
@@ -52,17 +52,17 @@
 
     @POST
     @Path("map-response")
-    @Consumes(APPLICATION_PROTOBUF)
-    @Produces(APPLICATION_PROTOBUF)
+    @Consumes(APPLICATION_X_PROTOBUF)
+    @Produces(APPLICATION_X_PROTOBUF)
     public Single<Response> postSingleResponse(@QueryParam("fail") final boolean fail,
                                                final Single<HelloRequest> single) {
         return postSingle(fail, single).map(m -> accepted(m).build());
     }
 
     @POST
     @Path("map-publisher")
-    @Consumes(APPLICATION_PROTOBUF)
-    @Produces(APPLICATION_PROTOBUF_VAR_INT)
+    @Consumes(APPLICATION_X_PROTOBUF)
+    @Produces(APPLICATION_X_PROTOBUF_VAR_INT)
```

Comment:
Do we need an opposite test?

```java
@Consumes(APPLICATION_X_PROTOBUF_VAR_INT)
@Produces(APPLICATION_X_PROTOBUF)
```

Reply:
This test lives in `PublisherProtobufResources`